dex/src/test/scala/com/wavesplatform/dex/settings/BaseSettingsSpecification.scala

```diff
@@ -12,6 +12,8 @@ class BaseSettingsSpecification extends AnyFlatSpec {
   def getSettingByConfig(conf: Config): Either[String, MatcherSettings] =
     Try(ConfigSource.fromConfig(conf).at("waves.dex").loadOrThrow[MatcherSettings]).toEither.leftMap(_.getMessage)
 
+  val correctSecureKeys: Set[String] = Set("user", "pass", "seed", "private", "java", "sun", "api")
```

Comment:
The values is equal to the default one. In these tests we use a different value.